src/contexts/Global/index.tsx

```diff
@@ -0,0 +1,294 @@
+import * as React from 'react'
+import PropTypes from 'prop-types'
+import Router from 'next/router'
+import { useRemoteConfig } from '../RemoteConfig'
+
+interface GlobalState {
+  /**
+   * A `boolean` which indicates whether the cart menu is open or not.
+   * The cart menu is the menu that appears when the user clicks on the cart icon.
+   * @default false
+   */
+  isCartMenuOpen: boolean
+  /**
+   * A `boolean` which indicates whether the cookie bar is open or not.
+   * The cookie bar is the bar that appears when the user enters the website
+   * for the first time.
+   *
+   * @note This value is set to `true` after a delay of `COOKIE_BAR_ENTER_DELAY`,
+   * provided the user has not previously dismissed the cookie bar.
+   * @default false
+   */
+  isCookieBarOpen: boolean
+  /**
+   * A `boolean` which indicates whether the market menu is open or not.
+   * The market menu contains the list of markets that the user can switch between.
+   * @default false
+   */
+  isMarketMenuOpen: boolean
+  /**
+   * A `boolean` which indicates whether the navigation menu is open or not.
+   * The navigation menu is the menu that appears when the user clicks on the
+   * three stacked lines icon, or otherwise known as the _hamburger_ icon.
+   * @default false
```

Comment:
> three stacked lines icon, or otherwise known as the _hamburger_ icon.

I'm afraid that this might be a bit too specific as well.
I'd want to opt for a more generic description, where the visual representation isn't tied to the documentation.